modules/mirror-registry-introduction.adoc

```diff
@@ -21,6 +21,11 @@ The following limitations apply to the _mirror registry for Red Hat OpenShift_:
 * The _mirror registry for Red Hat OpenShift_ is not a highly-available registry and only local file system storage is supported. It is not intended to replace {quay} or the internal image registry for {product-title}.
 
 * The _mirror registry for Red Hat OpenShift_ is only supported for hosting images that are required to install a disconnected {product-title} cluster, such as Release images or Red Hat Operator images. It uses local storage on your {op-system-base-full} machine, and storage supported by {op-system-base} is supported by the _mirror registry for Red Hat OpenShift_.
++
+[NOTE]
+====
+Because the _mirror registry for Red Hat OpenShift_ uses local storage, you should remain aware of storage usage consumed when mirroring images and employ {quay}'s garbage collection feature to mitigate potential issues. For more information about this feature, see "{quay} garbage collection".
```

Comment:
```suggestion
Because the _mirror registry for Red Hat OpenShift_ uses local storage, you should remain aware of the storage usage consumed when mirroring images and use {quay}'s garbage collection feature to mitigate potential issues. For more information about this feature, see "{quay} garbage collection".
```